styles.css

```diff
@@ -1 +1,69 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+body {
+    font-size: 28px;
+}
+
+.content {
+    display: grid;
+    background-color: #feefdb;
+    padding: 15px;
+    grid-gap: 16px;
+}
+
+.a, .b, .c, .d, .e, .f {
+    color: white;
+    background-color: #ba3b46;
+    display: grid;
+    flex-direction: row;
+    align-content: center;
+    justify-content: center;
+    margin: 0;
+}
+
+.a {
+    grid-area: a;
+}
+.b {
+    grid-area: b;
+    padding: 15px;
+}
+.c {
+    grid-area: c;
+}
+.d {
+    grid-area: d;
+}
+.e {
+    grid-area: e;
+}
+.f {
+    grid-area: f;
+}
+/* Mobile Screen */
+@media only screen and (max-width:800px) {
```

Comment:
Deze _@media_ had hier in principe niet gehoeven. Alleen _min-width_ is voldoende, aangezien het scherm dan ook zal veranderen wanneer de _width_ groter is dan 800px.